clients/js/README.md

```diff
@@ -28,6 +28,33 @@ import '@oasisprotocol/sapphire-hardhat';
 
 [@oasisprotocol/sapphire-hardhat]: https://www.npmjs.com/package/@oasisprotocol/sapphire-hardhat
 
+### Wagmi
+
+Try the [@oasisprotocol/sapphire-wagmi] package for your convenience.
+
+```ts
+import { createConfig } from 'wagmi';
+import { sapphireTestnet } from 'wagmi/chains';
+import {
+  injectedWithSapphire,
+  sapphireTransport,
+} from '@oasisprotocol/sapphire-wagmi';
+
+export const config = createConfig({
+  multiInjectedProviderDiscovery: false,
+  chains: [sapphireTestnet],
+  connectors: [injectedWithSapphire()],
+  transports: {
+    [sapphireLocalnet.id]: sapphireTransport(),
+  },
+});
+```
+
+Please note that [EIP-6963] (Multi Injected Provider Discovery) is not currently
+supported with Wagmi.
+
+[EIP-6963]: https://eips.ethereum.org/EIPS/eip-6963
+
```

Comment:
What? https://wagmi.sh/react/api/createConfig#multiinjectedproviderdiscovery

So `@oasisprotocol/sapphire-paratime` does not support `EIP-6963`? This reads like `Wagmi` does not support `EIP-6963`. Can we change the wording here a bit? So it's more clear.

Reply:
EIP-6963 is supported by Wagmi, however due to the way that MIPD is implemented in Wagmi I'm unable to find a way of wrapping the detected providers with Sapphire.

Reply:
I would just make it more clear that `@oasisprotocol/sapphire-paratime` does not support Wagmi's EIP-6963 implementation and you need to make a config with `multiInjectedProviderDiscovery` set to `false` in the dApp.

Reply:
This has been addressed in the README.md file from `integrations/wagmi-v2`